integration-test/common/src/test/java/org/apache/uniffle/test/ShuffleServerGrpcTest.java

```diff
@@ -626,4 +627,21 @@ private void addExpectedBlockIds(Roaring64NavigableMap bitmap, List<Long> blockI
       bitmap.addLong(blockIds.get(i));
     }
   }
+
+  @Test
+  public void testGettingUserWhenUsingHdfsStorage() throws Exception {
```

Comment:
Could we add some Spark integration tests?

Reply:
Yes. After I solve the above problems, i will attach more test cases.